tests/foreman/cli/test_provisioningtemplate.py

```diff
@@ -147,47 +115,35 @@ def test_positive_update_with_manager_role(module_location, module_org):
     :CaseImportance: Medium
 
     :BZ: 1277308
+
+    :parametrized: yes
     """
     new_name = gen_string('alpha')
     username = gen_string('alpha')
     password = gen_string('alpha')
-    template = make_template(
+    template = module_target_sat.cli_factory.make_template(
         {'organization-ids': module_org.id, 'location-ids': module_location.id}
     )
-    # Create user with Manager role
-    user = entities.User(
+    # Create user with Manager/Organization admin role
+    user = module_target_sat.api.User(
         login=username,
         password=password,
         admin=False,
         organization=[module_org.id],
         location=[module_location.id],
     ).create()
-    User.add_role({'id': user.id, 'role': "Manager"})
+    module_target_sat.cli.User.add_role({'id': user.id, 'role': role_name})
     # Update template name with that user
-    Template.with_user(username=username, password=password).update(
+    module_target_sat.cli.Template.with_user(username=username, password=password).update(
         {'id': template['id'], 'name': new_name}
     )
-    template = Template.info({'id': template['id']})
+    template = module_target_sat.cli.Template.info({'id': template['id']})
     assert new_name == template['name']
 
 
 @pytest.mark.tier1
```

Comment:
This removed test case is not mentioned in description. Is it intentionally removed? If yes, then please add it into description

Reply:
it is already mentioned `Removing redundant tests which are covered already either in e2e or in some other tests.`